os/board/rtl8721csm/src/Makefile

```diff
@@ -142,13 +166,32 @@ CFLAGS += -I$(COMPONENT_DIR)/common/drivers/wlan/realtek/wlan_ram_map/rom
 CFLAGS += -I$(COMPONENT_DIR)/os/os_dep/include
 CFLAGS += -I$(COMPONENT_DIR)/os/tizenrt
 CFLAGS += -I$(TOPDIR)/net/lwip
-CFLAGS += -I$(TOPDIR)/net/lwip/src/include/
+CFLAGS += -I$(TOPDIR)/net/lwip/src/include
```

Comment:
Duplicated. Let's remove.

Reply:
Do you mean to remove `CFLAGS += -I$(TOPDIR)/net/lwip/src/include` this line?

Reply:
Right.

Reply:
It has been removed.